src/ansys/fluent/core/solver/flobject.py

```diff
@@ -228,7 +228,7 @@ def flproxy(self):
         return self._flproxy
 
     @property
-    def file_transfer_service(self):
+    def _file_transfer_handler(self):
```

Comment:
@seanpearsonuk 

We need to rename `file_transfer_service` to `__file_transfer_handler` as I already mentioned in the issue description that we are setting `_file_transfer_service` on root object therefore we can not use same name. Tests are passing now.